### DISCUSSION TITLE
Restart counts of container's pod not reflected in the API

### DISCUSSION THREAD

**Discussion by Garahk** [netdata/netdata#14865] [Q&A] [answered]
Checking the API, and a particular pod, the number of restarts indicate 0, although it would seem the metric it's restarts per second (restarts/s)?

![image](https://user-images.githubusercontent.com/99432183/230218911-0ebac34f-9a59-4f17-bd77-250778256ded.png)

However on console, via kubeclt, the pod seems to have had at least 3 restarts.

![image](https://user-images.githubusercontent.com/99432183/230218958-e5a756b2-7310-41e8-bb36-b07c5e16a875.png)

Is the metric reflected by netdata's API supposed to be the same as restart counts? Or are we seeing here a bug?

**Comment by ilyam8** [comment, marked as answer]
Hi, @Garahk. I checked and you are right - Netdata shows restarts per second. I think changing it to the actual number of restarts makes sense.

I changed restarts rate to count in https://github.com/netdata/go.d.plugin/pull/1141

**Reply by Garahk** [reply]
Hello @ilyam8, sounds good, thank you for replying so quick.
